bin/tass

```diff
@@ -57,10 +57,17 @@ when 'create'
   end
 
   config = initialize_environment(opts)
+  sfr_config = {}  # @TODO: remove this when SFR config becomes sane
 
   # check the verison of the yaml we are reading, and do the new hotness if we are running v2 or higher
   if newer_yaml_version?(config)
     # Determine if we are making a cloudformation request
+    if config[:stack][:spot_fleet]
+      sfr_config.merge!(config[:stack][:spot_fleet][:config_data])
+      config[:stack].delete(:spot_fleet)
+      Tapjoy::AutoscalingBootstrap::EC2.request_spot_instances(sfr_config,
+        config[:userdata_dir])
```

Comment:
we found that we can't capture the id via cloudformation, and so needed to make an exception for spotfleets